var/spack/repos/builtin/packages/py-pyproj/package.py

```diff
@@ -9,19 +9,24 @@
 class PyPyproj(PythonPackage):
     """Python interface to the PROJ.4 Library."""
 
-    homepage = "http://jswhit.github.io/pyproj/"
-    url      = "https://github.com/jswhit/pyproj/tarball/v1.9.5.1rel"
-    git      = "https://www.github.com/jswhit/pyproj.git"
+    homepage = "https://github.com/pyproj4/pyproj"
+    url      = "https://pypi.io/packages/source/p/pyproj/pyproj-2.2.0.tar.gz"
+    git      = "https://github.com/pyproj4/pyproj.git"
 
-    # This is not a tagged release of pyproj.
-    # The changes in this "version" fix some bugs, especially with Python3 use.
-    version('1.9.5.1.1', commit='0be612cc9f972e38b50a90c946a9b353e2ab140f')
-    version('1.9.5.1', 'a4b80d7170fc82aee363d7f980279835')
+    maintainers = ['adamjstewart', 'citibeth']
```

Comment:
Can you list me first, since I was the original author?  I don't have a burning need to control all possible changes.  If somebody suggests changes to this package, I'd want to have the chance to review it, in case it inadvertently breaks something I rely upon.